From 2deb458c5fd4ac318018b8464fa677dc4570ba61 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Tue, 16 May 2023 16:34:31 +0900
Subject: [PATCH] udev: use SYNTHETIC_ERRNO() at one more place

(cherry picked from commit b3cfe5900108df81fbf547b297d51ac8c7359a9b)

Related: RHEL-5988
---
 src/udev/udevadm-test-builtin.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/udev/udevadm-test-builtin.c b/src/udev/udevadm-test-builtin.c
index 81b633611e..5570eec789 100644
--- a/src/udev/udevadm-test-builtin.c
+++ b/src/udev/udevadm-test-builtin.c
@@ -87,8 +87,7 @@ int builtin_main(int argc, char *argv[], void *userdata) {
 
         cmd = udev_builtin_lookup(arg_command);
         if (cmd < 0) {
-                log_error("Unknown command '%s'", arg_command);
-                r = -EINVAL;
+                r = log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Unknown command '%s'", arg_command);
                 goto finish;
         }
 
